test/README.md

```diff
@@ -36,13 +36,13 @@ To conduct tests for the IBC integration system, follow these steps:
 
 Before initiating the tests, configure essential environment variables:
 
-- **`E2E_CONFIG_PATH`**: Set this variable to the absolute path of your chosen configuration file. You can create these configuration files using the sample files provided in the `centralised-relay` source folder. Sample configuration files are available at the following locations:
+- **`TEST_CONFIG_PATH`**: Set this variable to the absolute path of your chosen configuration file. You can create these configuration files using the sample files provided in the `centralised-relay` source folder. Sample configuration files are available at the following locations:
```

Comment:
Should `centralised-relay` be `centralized-relay`?

Reply:
updated.